charts/hivemq-platform/tests/services/hivemq_control_center_services_test.yaml

```diff
@@ -132,7 +132,7 @@ tests:
           hivemqProxyProtocol: true
     asserts:
       - failedTemplate:
-          errorMessage: Service type control-center with container port 8081 is using PROXY protocol value. PROXY protocol is only supported for MQTT and WebSocket services
+          errorPattern: Service type `control-center` with container port `8081` is using `hivemqProxyProtocol` value. HiveMQ PROXY protocol is only supported by MQTT and WebSocket services
```

Comment:
Why is it necessary to give the container port in this message. Should not be needed.